pkg/virtctl/imageupload/imageupload.go

```diff
@@ -177,6 +179,22 @@ func parseArgs(args []string) error {
 		return fmt.Errorf("cannot upload to a readonly volume, use either ReadWriteOnce or ReadWriteMany if supported")
 	}
 
+	if volumeMode != "" {
+		if volumeMode != string(v1.PersistentVolumeBlock) && volumeMode != string(v1.PersistentVolumeFilesystem) {
+			return fmt.Errorf("%s is not supported for volumeMode, use either Block or Filesystem!", volumeMode)
```

Comment:
Better use `string(v1.PersistentVolumeBlock)` instead of spelling out `Block` to be sure we do not have a typo.
I would drop the exclamation mark; there is no matter of urgency or alarm.

Reply:
Done.